insert-special-characters.php

```diff
@@ -17,6 +17,53 @@
 
 namespace InsertSpecialCharacters;
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @since 1.0.8
+ *
+ * @return string Minimum version required.
+ */
+function minimum_php_requirement(): string {
```

Comment:
```suggestion
function minimum_php_requirement() {
```

The PHP version check needs to run in PHP 5.6+ (as we support pre 6.3 versions of WP).

Reply:
Done